src/clientsettingsdlgbase.ui

```diff
@@ -1083,6 +1083,33 @@
                </property>
               </spacer>
              </item>
+             <item>
+              <widget class="QLabel" name="lblDetectFeedback">
```

Comment:
There needs to be "What's this?" text for the label and the control along with accessibility text for the control.